tests/appsec/test_remoteconfiguration.py

```diff
@@ -1,16 +1,30 @@
+import json
 import os
+import time
 
+import mock
 import pytest
 
+from ddtrace.appsec import _asm_context
 from ddtrace.appsec._remoteconfiguration import appsec_rc_reload_features
+from ddtrace.appsec.utils import _appsec_rc_capabilities
 from ddtrace.appsec.utils import _appsec_rc_features_is_enabled
 from ddtrace.constants import APPSEC_ENV
 from ddtrace.constants import APPSEC_JSON
 from ddtrace.contrib.trace_utils import set_http_meta
 from ddtrace.ext import SpanTypes
+from ddtrace.internal import _context
+from ddtrace.internal.remoteconfig import RemoteConfig
+from tests.appsec.test_processor import Config
 from tests.utils import override_env
 
 
+@pytest.fixture
+def remote_config_worker(tracer):
+    RemoteConfig._worker = None
```

Comment:
do you need to reset this after the fixture?

```python
old_worker = RemoteConfig._worker
RemoteConfig._worker = None
try:
    yield
finally:
    RemoteConfig._worker = old_worker
```

Reply:
also... why set it to `None` ?

Reply:
Remote Config Worker has an initial state of None when the application start. We want to use a "clean" Remote Config worker for each unit test